aggregator/handler/handler.go

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2017 Uber Technologies, Inc.
+// Copyright (c) 2018 Uber Technologies, Inc.
```

Comment:
rename to types.go?

Reply:
There's already a `type.go` file defining different handler types hence the `handler.go`.